editor/src/components/navigator/layout-element-icons.ts

```diff
@@ -205,7 +205,10 @@ function createComponentIconProps(
   const elementName = MetadataUtils.getJSXElementName(path, components, metadata.components)
   const element = TP.isInstancePath(path)
     ? MetadataUtils.getElementByInstancePathMaybe(metadata.elements, path)
```

Comment:
Could slightly rewrite this to have the ternary produce the value which is passed to `getElementByInstancePathMaybe`.